hack/ginkgo-e2e.sh

```diff
@@ -191,7 +191,7 @@ fi
   --dns-domain="${KUBE_DNS_DOMAIN:-cluster.local}" \
   --ginkgo.slowSpecThreshold="${GINKGO_SLOW_SPEC_THRESHOLD:-300}" \
   ${CONTAINER_RUNTIME:+"--container-runtime=${CONTAINER_RUNTIME}"} \
-  ${MASTER_OS_DISTRIBUTION:+"--master-os-distro=${MASTER_OS_DISTRIBUTION}"} \
+  ${CONTROL_PLANE_OS_DISTRIBUTION:+"--control-plane-os-distro=${CONTROL_PLANE_OS_DISTRIBUTION}"} \
```

Comment:
this change has a bit of a chicken and an egg issue, it will require other changes ready to go: https://cs.k8s.io/?q=MASTER_OS_DISTRIBUTION&i=nope&files=&excludeFiles=&repos=

Alternatively we could update the logic here to support both names, and then start to migrate the CI etc. to use the new name

Reply:
Just saw this comment @BenTheElder. I could make the PRs, but I'm not sure how coordination would go since PRs may take awhile to get approved in any one of those areas and we'd probably want to have them merged simultaneously.

I'll update this to support both names.

Reply:
Sorry it seems I missed replying to this thread :(

I suspect the most viable option is to accept both environment variables here in the bash in the first PR, then we can update the places setting it to set both, and eventually delete the old variable name from both.

kubeadm is going through a similar change here in https://github.com/kubernetes/kubeadm/issues/2200